dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataAPI.kt

```diff
@@ -69,9 +52,10 @@ interface DataAPI<T> {
         produces = ["application/json"]
     )
     /**
-     * A method to retrieve a specific data set identified by its ID
-     * @param dataId identifier used to uniquely determine the data set in the data store
-     * @return the complete data stored under the provided ID
+     * A method to retrieve specific data identified by its ID
+     * @param dataId identifier used to uniquely specify data in the data store
+     * @return the complete data stored under the provided data ID with the associated company ID
      */
-    fun getCompanyAssociatedDataSet(@PathVariable("dataId") dataId: String): ResponseEntity<CompanyAssociatedDataSet<T>>
+    fun getCompanyAssociatedDataSet(@PathVariable("dataId") dataId: String):
```

Comment:
Warum heißt es postCompanyassociatedData, aber getCompanyAssosciatedData**Set**?

Reply:
Ich denke beides muss CompanyAssociatedData heißen.   Denn in beiden Fällen bekommt man Data + Company.
Ändere ich jetzt so

Reply:
Fixed

Reply:
ok